atd-vzd/migrations/default/1724353502869_remove_legacy_objects/up.sql

```diff
@@ -0,0 +1,189 @@
+drop table if exists atd_txdot__rpt_road_type_lkp;
+drop table if exists atd_txdot__rural_urban_lkp;
+drop table if exists atd_txdot__rural_urban_type_lkp;
+drop table if exists atd_txdot__rwy_sys_lkp;
+drop table if exists atd_txdot__shldr_type_lkp;
+drop table if exists atd_txdot__shldr_use_lkp;
+drop table if exists atd_txdot__specimen_type_lkp;
+drop table if exists atd_txdot__speed_mgmt_lkp;
+drop table if exists atd_txdot__state_lkp;
+drop table if exists atd_txdot__street_sfx_lkp;
+drop table if exists atd_txdot__substnc_cat_lkp;
+drop table if exists atd_txdot__substnc_tst_result_lkp;
+drop table if exists atd_txdot__surf_cond_lkp;
+drop table if exists atd_txdot__surf_type_lkp;
+drop table if exists atd_txdot__traffic_cntl_lkp;
+drop table if exists atd_txdot__trauma_centers_lkp;
+drop table if exists atd_txdot__tst_result_lkp;
+drop table if exists atd_txdot__unit_desc_lkp;
+drop table if exists atd_txdot__unit_dfct_lkp;
+drop table if exists atd_txdot__uom_lkp;
+drop table if exists atd_txdot__veh_color_lkp;
+drop table if exists atd_txdot__veh_damage_description_lkp;
+drop table if exists atd_txdot__veh_damage_severity_lkp;
+drop table if exists atd_txdot__veh_direction_of_force_lkp;
+drop table if exists atd_txdot__veh_make_lkp;
+drop table if exists atd_txdot__veh_mod_lkp;
+drop table if exists atd_txdot__veh_trvl_dir_lkp;
+drop table if exists atd_txdot__veh_year_lkp;
+drop table if exists atd_txdot__wdcode_lkp;
+drop table if exists atd_txdot__wthr_cond_lkp;
+drop table if exists atd_txdot__y_n_lkp;
+drop table if exists atd_txdot__yes_no_choice_lkp;
+drop table if exists atd_txdot__est_comp_cost;
+drop table if exists atd_txdot__est_comp_cost_crash_based;
+drop table if exists atd_txdot__est_econ_cost;
+drop table if exists atd_txdot_change_log;
+drop table if exists atd_txdot_change_pending;
+drop table if exists atd_txdot_change_status;
+drop table if exists atd_txdot_changes;
+drop view if exists atd_txdot_changes_view;
+drop table if exists atd_txdot_cities;
+drop view if exists atd_txdot_crash_locations_ranking;
+drop table if exists atd_txdot_crash_status;
+drop table if exists atd_txdot_geocoders;
+drop table if exists atd_txdot_locations_change_log;
+drop table if exists atd_txdot_streets;
+drop table if exists costs_list;
+drop view if exists locations_with_crash_injury_counts;
+drop view if exists grafana_crash_counts;
+drop table if exists hin_corridors;
+drop table if exists intersection_road_map;
+drop table if exists intersections;
+drop table if exists polygons;
+drop view if exists view_location_crashes_by_manner_collision;
+drop view if exists view_location_injry_count_cost_summary;
+drop view if exists view_vzv_by_mode;
+drop view if exists view_vzv_by_month_year;
+drop view if exists view_vzv_by_time_of_day;
+drop view if exists view_vzv_demographics_age_sex_eth;
+drop view if exists view_vzv_header_totals;
+drop table if exists cr3_mainlanes;
+drop table if exists non_cr3_mainlanes;
```

Comment:
@johnclary i was skimming through these and this one caught my eye. If the way we calculate non-CR3 locations hasn't change, then I think that we need to keep this table since it is referenced in `update_noncr3_location()` which is used in the trigger on insert or update of records in the `atd_apd_blueform` table.

Reply:
such a fantastic catch—thank you. i guess i really need to test the afd + ems in addition to the blueform import. i will do that and strip out this table deletion 🙏

Reply:
thanks again @mddilley — we will not delete `non_cr3_mainlanes` ✅  i went through all of the trigger functions in the DB again to double check i didn't miss anything else.